src/core/Table/utils.ts

```diff
@@ -6,14 +6,20 @@ import { ColumnInstance } from 'react-table';
 
 export const getCellStyle = <T extends Record<string, unknown>>(
   column: ColumnInstance<T>,
+  isTableResizing = false,
 ): React.CSSProperties | undefined => {
   const style = {} as React.CSSProperties;
   style.flex = `1 1 145px`;
   if (column.width) {
     const width =
       typeof column.width === 'string' ? column.width : `${column.width}px`;
     style.width = width;
-    style.flex = `0 0 ${width}`;
+    // This allows flexbox to handle the width of the column on table resize
+    if (isTableResizing && column.canResize) {
+      style.flex = `${Number(column.width)} 1 ${width}`;
+    } else {
+      style.flex = `0 0 ${width}`;
```

Comment:
Would it be bad if we always set the flex to column width even if it's not resizable?

Reply:
Yup because we would need to always handle table resize in js.